src/dmd/glue.d

```diff
@@ -1244,6 +1244,11 @@ void FuncDeclaration_toObjFile(FuncDeclaration fd, bool multiobj)
     if (fd.isExport())
         objmod.export_symbol(s, cast(uint)Para.offset);
 
+    if (fd.isCrtCtorDtor & 1)
+        objmod.setModuleCtorDtor(s, true);
+    if (fd.isCrtCtorDtor & 2)
+        objmod.setModuleCtorDtor(s, false);
```

Comment:
When using magic numbers, please document them at the definition of isCrtCtorDtor at a minimum.

Reply:
I see you did. Never mind.